pallets/parachain-staking/src/lib.rs

```diff
@@ -729,120 +687,99 @@ pub mod pallet {
 		pub fn cancel_leave(&mut self) {
 			self.status = DelegatorStatus::Active
 		}
-		pub fn add_delegation(&mut self, bond: Bond<AccountId, Balance>) -> bool {
+		pub fn add_delegation(&mut self, bond: Bond<A>) -> bool {
 			let amt = bond.amount;
+			let liquidity_token = bond.liquidity_token;
```

Comment:
Does this mean nominators can delegate only the same token as staked by the collator?

Reply:
Delegators when delegating to a certain collator must use the same token. But a delegator may delegate to various collators using the corresponding token.
A collator however has to decide which token he wants to use, and has to leave then rejoin to change his token (dropping/undoing his associated delegations).